pym/bob/scm/git.py

```diff
@@ -278,14 +281,17 @@ def status(self, workspacePath, dir, verbose=0):
             ret = 'clean'
             if status == "":
                 if verbose >= 3:
-                    print(colorize("   STATUS      {0}".format(scmdir), "32"))
+                    print(colorize("   STATUS {0}     {1}".format(overriddenStatus, scmdir), "32"))
             elif status != "u":
                 ret = 'unclean'
 
             if (status != "") and (verbose != 0):
-                print(colorize("   STATUS {0: <4} {1}".format(status, scmdir), "33"))
+                print(colorize("   STATUS {0}{1: <4} {2}".format(overriddenStatus, status, scmdir), "33"))
```

Comment:
Well in a previous version there was a `STATUS O` output if verbose >= 1. But a overridden SCM is still in clear state if it's only overridden and not modified - that’s why I think bob status shouldn't show a different behavior as for non overridden clean SCMs. Also if you have a override matching to all of you SCM's (maybe because you're using a local mirror) you'll get a lot of STATUS 0 prints.